questionnaires/chair_rises/chair_rises_armt.json

```diff
@@ -0,0 +1,403 @@
+[
+    {
+        "field_name": "chair_rises_instructions",
+        "form_name": "chair_rises",
+        "section_header": "",
+        "field_type": "info",
+        "field_label": "Chair Rises Instructions",
+        "select_choices_or_calculations": [
+            {
+                "code": "Part1",
+                "label": "Comfortable clothing and appropriate footwear should be worn for this test."
+            },
+            {
+                "code": "Part2",
+                "label": "Sit on a straight back chair (for additional stability, place the rear legs of the chair against a wall so that it cannot slide backwards)."
+            },
+            {
+                "code": "Part3",
+                "label": "Press button A (activity button A is located at the top right-hand side of the watch face)"
+            },
+            {
+                "code": "Part4",
+                "label": "Note! If this is the first time you have started an activity, select the checkbox next to 'Walk' and 'Cardio' to add these activities to your favorites."
```

Comment:
There was also video instructions. should we add them as a descriptive input too?